ngraph/python/tests/test_ngraph/test_reduction.py

```diff
@@ -104,15 +105,16 @@ def test_non_max_suppression():
 
     boxes_shape = [1, 1000, 4]
     scores_shape = [1, 1, 1000]
-    expected_shape = [0, 3]
     boxes_parameter = ng.parameter(boxes_shape, name="Boxes", dtype=np.float32)
     scores_parameter = ng.parameter(scores_shape, name="Scores", dtype=np.float32)
 
     node = ng.non_max_suppression(boxes_parameter, scores_parameter)
 
     assert node.get_type_name() == "NonMaxSuppression"
-    assert node.get_output_size() == 1
-    assert list(node.get_output_shape(0)) == expected_shape
+    assert node.get_output_size() == 3
```

Comment:
Is it expected that we changed this test?

Reply:
Why this test should be changed?

Reply:
Is it question to you 😄 Why did you change the nGraph Python test? And how did it work before?

Reply:
I changed this test because NMS-5 has 3 outputs. Otherwise, this test failed for NMS-5.

Reply:
But why we changed this test? Am I right that these tests before used NMS-4?
I think that we still should have tests for old operations. Could you add a separate test for NMS -5?

Reply:
This test uses NMS from default opset, and `default_opset = 'opset5'`. If we want to run test for NMS-4 also, we need the function that returns the version of the operation. Are there such function, i.e. a function like `get_opset()` from MO?